### ISSUE TITLE
`pooltogether-rng-contracts@1.5.1` fails some tests when `yarn.lock` is removed

### ISSUE THREAD

**Issue by nikola-matic** [pooltogether/v4-core#287]
As part of our Solidity CI, we have a suite of external tests (including v4-core), where we build/compile these external projects with the latest compiler. One of the prerequisites is to remove any `*.lock` files, and then go through the build procedure.
As of the recent `v1.4.0` bump, the step for `v4-core` started failing, which we managed to pinpoint on the `rng-contracts` bump from `1.4.0` to `1.5.1`. There are a total of 8 failing tests, due to:
```
  1) Oracle jobs
       "before each" hook for "should be able to trigger the beacon":
     ERROR processing /tmp/ext-test-Pool-Together-V4-2Cs30p/ext/deploy/deploy.js:
Error: cannot find artifact "RNGServiceStub"
```
and
```
  3) PrizePool
       "before all" hook in "PrizePool":
     Error: cannot find artifact "ERC20Mintable"
```
#### Steps to reproduce
1. clone `v4-core` repository
2. `git checkout master` (should be default)
3. `rm yarn.lock`
4. `yarn`
5. `mkdir abis` in repo root (this is probably optional, but the only way I actually got the tests to run)
6. `yarn test`

If the `pooltogether-rng-contracts` is downgraded to `1.4.0` in `package.json`, the tests pass, which is the approach we'll take until this is fixed (or not).

**Comment by asselstine** [comment]
@nikola-matic Thank you for reporting this; we'll take a look

**Comment by PierrickGT** [comment]
@nikola-matic thanks for reporting this issue.

Artifacts from the `pooltogether-rng-contracts` package are now exported in the `artifacts` folder and not the `build` one.
I've fixed it in this commit to include these files in the package: https://github.com/pooltogether/pooltogether-rng-contracts/commit/0263fd6e74375c18442c8dca502ec9980f0f7c3c
And released v1.5.2: https://github.com/pooltogether/pooltogether-rng-contracts/releases/tag/v1.5.2

I've also fixed `v4-core` in this commit: https://github.com/pooltogether/v4-core/commit/86e64fe3c35927f95c1405a7844bb4f2dc3739d2
And released v1.3.1: https://github.com/pooltogether/v4-core/releases/tag/v1.3.1

Both packages are available on NPM.
